actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -90,12 +90,17 @@ module DateHelper
       #   distance_of_time_in_words(from_time, from_time + 50.minutes, scope: 'datetime.distance_in_words.short') # => "an hour"
       #   distance_of_time_in_words(from_time, from_time + 3.hours, scope: 'datetime.distance_in_words.short')    # => "3 hours"
       def distance_of_time_in_words(from_time, to_time = 0, options = {})
+        if [from_time, to_time].any?(&:nil?)
+          raise ArgumentError, "given time must not be nil"
+        end
+
         options = {
           scope: :'datetime.distance_in_words'
         }.merge!(options)
 
         from_time = from_time.to_time if from_time.respond_to?(:to_time)
         to_time = to_time.to_time if to_time.respond_to?(:to_time)
+        from_time, to_time = Time.at(from_time), Time.at(to_time)
```

Comment:
I believe this is effective a conversion of input to Time. One thing to note is that it's not timezone aware. The suite seemed happy locally so maybe it's not a concern since this method deals in relative times? Truth is I don't trust myself with time, so I'd love a sanity check on this...